meson.build

```diff
@@ -150,8 +156,9 @@ subdir('examples')
 pkgconfig = import('pkgconfig')
 pkgconfig.generate(
 	libraries: lib_wlr,
-	version: '0.0.1',
-	filebase: 'wlroots',
-	name: 'wlroots',
+	subdirs: '@0@-@1@'.format(meson.project_name(), meson.project_version()),
+	version: meson.project_version(),
+	filebase: meson.project_name(),
+	name: meson.project_name(),
 	description: 'Wayland compositor library',
 )
```

Comment:
-1 to this commit. I'd rather not try to support several wlroots versions installed side-by-side.

Reply:
Well, it's less about supporting parallel installs of different versions.  The sole purpose of that part is just to have the installed include-tree in an isolated directory, so it won't (possibly) clash with the include-tree in the source-dir.

Reply:
Still not convinced. I don't like it.

Reply:
So how should that be changed?  Just keeping `meson.project_name()` for the subdir?

Reply:
Aye.

Reply:
Fixed.